File: NameApp/app/src/main/java/kevin/codelab/nameapp/MainActivity.java
package kevin.codelab.nameapp;

import android.os.Bundle;
import android.widget.TextView;
import android.widget.Toast;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import kevin.codelab.nameapp.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    ActivityMainBinding binding;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        super.onCreate(savedInstanceState);
        setContentView(binding.getRoot());


        binding.tvText1.setOnClickListener(v ->
                Toast.makeText(this,"Clicked Text one", Toast.LENGTH_LONG).show()
        );

        binding.tvText2.setOnClickListener(v ->
                Toast.makeText(this,"Clicked Text Two", Toast.LENGTH_LONG).show()
        );
    }
}

File: QuizApp/app/src/main/java/kevin/codelab/quizapp/MainActivity.kt
package kevin.codelab.quizapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kevin.codelab.quizapp.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        binding = ActivityMainBinding.inflate(layoutInflater)
        super.onCreate(savedInstanceState)
        setContentView(binding.root)


        binding.btnStart.setOnClickListener {
            if (binding.etName.text!!.isEmpty()){
                Toast.makeText(this,
                    "Please enter your name", Toast.LENGTH_LONG).show()
            } else {
                val intent = Intent(this, QuizQuestionsActivity::class.java)
                startActivity(intent)
                finish()
            }
        }
    }
}